public/wakeUp.js

```diff
@@ -0,0 +1,15 @@
+window.addEventListener("focus", () => {
```

Comment:
wakeup.js or wake_up.js

usually, we do have app/javascript for js files in rails applications.

but it's a special case here since we usually use ember for frontend stuff, but not for the login page. 

maybe add this to a sub folder inside public? isn't it a bit messy like this?